src/main/java/uk/gov/hmcts/divorce/caseworker/service/task/GenerateCoversheet.java

```diff
@@ -41,4 +41,22 @@ public void generateCoversheet(final CaseData caseData,
             formatDocumentName(caseId, COVERSHEET_DOCUMENT_NAME, now(clock))
         );
     }
+
+    public void generateCoversheet(final CaseData caseData,
+                                   final Long caseId,
+                                   final String templateId,
+                                   final Map<String, Object> templateContent,
+                                   final LanguagePreference languagePreference,
+                                   final String filename) {
+        log.info("Generating coversheet {} for sole case id {} ", filename, caseId);
```

Comment:
not sure this is called only for sole application ?

Reply:
I can change this to not say sole, thanks!